File: documentation-examples-scala/src/scala/dev/typr/foundationssc/docs/postgresql/XmlType.scala
package dev.typr.foundationssc.docs.postgresql
import dev.typr.foundationssc.*
import dev.typr.foundationssc.data.*



@SuppressWarnings(Array("unused"))
object XmlType:
  //start
  val xmlType: PgType[Xml] = PgTypes.xml
  val doc: Xml = new Xml("<root><child>text</child></root>")
  //stop

File: documentation-examples-kotlin/src/kotlin/dev/typr/foundationskt/docs/postgresql/XmlType.kt
package dev.typr.foundationskt.docs.postgresql

import dev.typr.foundationskt.*
import dev.typr.foundationskt.data.*

@Suppress("unused")
class XmlType {
    //start
    val xmlType: PgType<Xml> = PgTypes.xml
    val doc: Xml = Xml("<root><child>text</child></root>")
    //stop
}
